### PR TITLE
Add support of numeric values of mods

### PR DESCRIPTION
**Pull request by eGavr**
It has appeared that mods can have numeric values :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ function normalize(dep, options) {
             });
         } else {
             Object.keys(dep.mods).forEach(function(mod) {
-                if (typeof dep.mods[mod] === 'string' || typeof dep.mods[mod] === 'boolean') {
+                if (!Array.isArray(dep.mods[mod])) {
```
Comment (iamstarkov) [inline index.js]: why did you remove all the previous checks?

**Comment by iamstarkov** [comment]
Thanks for pull-request!

**Comment by iamstarkov** [comment]
looks good and reasonable, but Im confused with checks on line 66

File: index.js (hunk 1 above)
Comment (eGavr) [inline index.js]: seems to be the same. If you do not like,  I can return the full condition
